frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -193,9 +194,10 @@ const getAvailableFilters = (initialFilters): PageFilters => {
   return filters;
 };
 
+const filterPreference = ['kind'];
+const defaultActiveFilters = { kind: ['ClusterServiceVersion'] };
```

Comment:
Why are you introducing a new way to identify default enabled filters when this is already present in `getAvailableFilters`.

Reply:
Because, in the existing implementation, the filters are additive in nature.
If you use /catalog/ns/bar?kind=%5B%22Template%22%5D it will select both `clusterServiceVersion` (defaultFilter)  and `template` passed by URL. 
However, I am moving back to the existing implementation of defaultfilters.

Reply:
Be aware that until recently, all filters were always OFF by default. Therefore we wouldn't have any of these issues.